packages/manager/apps/catalog-revamp/src/hooks/useCatalog.ts

```diff
@@ -0,0 +1,67 @@
+import { useState, useEffect } from 'react';
+import { useQuery, UseQueryResult } from '@tanstack/react-query';
+import {
+  getManagerHubCatalogList,
+  getManagerHubCatalogListQueryKey,
+} from '@/api';
+import {
+  groupItems,
+  filterByEnum,
+  getAvailableCategories,
+  matchCategories,
+  matchUniverses,
+  matchSearchText,
+  filterItems,
+  getUniverses,
+} from '@/utils/utils';
+
+export type Product = {
+  id: number;
+  name: string;
+  description: string;
+  lang: string;
+  categories: string[];
+  url: string;
+  regionTags: string[];
+  productName: string;
+  order: string;
+  universe: string;
+  category: string;
+};
+
+interface ServiceData {
+  status: number;
+  data: any;
+}
+
+interface CatalogProps {
```

Comment:
```suggestion
interface UseCatalogParams {
```

Usually we call props the inputs of a component. Here it is a hook so parameters or params is better.

Reply:
Done